Havit.Blazor.Components.Web.Bootstrap/TreeViews/HxTreeViewItem.razor.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.ComponentModel;
+using Microsoft.AspNetCore.Components;
+
+namespace Havit.Blazor.Components.Web.Bootstrap
+{
+	public partial class HxTreeViewItem<TValue> : ComponentBase
+	{
+		[Parameter] public ValueWrapper<TValue> Value { get; set; }
+
+		private string PaddingLeft { get; set; }
+
+		protected override void OnInitialized()
+		{
+			if (Value == null)
+			{
+				return;
+			}
+
+			PaddingLeft = $"{Value.Level * 15}px";
```

Comment:
@sshulik This needs to be customizable in some way.
Not sure whether we should set the left padding explicitly or whether to use some kind of nested containers for the tree-structure and let every container to add some more padding to the previous one. This might allow leaving the padding configurable with CSS without having to compute and set any value from code.

@crdo, can you please add your opinion here?